client/src/components/FollowList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+import { Modal, ModalHeader, ModalBody, Button } from 'reactstrap';
+import userphoto from '../../assets/user1.png';
+import './style.scss';
+interface IFollowData 
+ {
+    username: string;
+}
```

Comment:
interface IFollowData {
    username: string;
}

because hard to read

Reply:
fixed